data/mods/blazemod/blaze_override.json

```diff
@@ -258,5 +258,24 @@
             {"item": "steel_chunk", "min": 5, "max": 10},
             {"item": "scrap", "min": 5, "max": 10}
         ]
+    },{
```

Comment:
Does this entry actually change anything from the usual battery mount? If not, it should be removed.